backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/ToplevelPhases.kt

```diff
@@ -275,6 +284,7 @@ internal val toplevelPhase = namedUnitPhase(
         name = "Compiler",
         description = "The whole compilation process",
         lower = frontendPhase then
+                objCExportPhase then
```

Comment:
This is an interesting problem, if the pipeline to be built conditionally for each platform or include noop phases like this one. @alexander-gorshenev WDYT?

Reply:
Unfortunately, header generation phase is part of frontend. Luckily, this phase doesn't affect IR.
Btw, frontend is already different for different backends.